packages/gatsby-core-utils/src/__mocks__/ci-info.ts

```diff
@@ -1,13 +1,13 @@
-const ciInfo = jest.genMockFromModule(`ci-info`)
+const ciInfo: typeof import("ci-info") = jest.genMockFromModule(`ci-info`)
```

Comment:
is this valid? typically `import()` are async. So this seems off. /shrug

Reply:
I think you're referring to Webpack's `import()` function that does async import. In TypeScript you can pull in the export signature this way.